default.yaml

```
@@ -167,6 +167,16 @@ onChainWallet:
   scanDepthOutgoing: 2
   scanDepthChannelUpdate: 8
 
+swap:
+  minOnChainHotWalletBalance: 30000000
+  swapOutAmount:  25000000
```

```suggestion
  minOnChainHotWalletBalance: 50000000
  swapOutAmount:  50000000
```

probably better default